safe-global/safe-wallet-web
# fix: prevent recovery with same setup

## What it solves

Prevents [recovery attempts with the same setup](https://www.notion.so/safe-global/Error-when-creating-editing-recovery-setup-without-changes-f82ad44386e547c99d2a9444854e4344)

## How this PR fixes it

This prevents recovery attempts from being created with the same owner/threshold setup. All owner/threshold fields are dependent on each other and validate together if the setup is the same as the current one.

## How to test it

Create a recovery attempt, proposing the same owners and threshold. Observe the validation error preventing futher progression.

## Screenshots

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/c478090f-0f88-4eba-9a24-902053607621)

## Checklist
